core/test/gather/gatherers/trace-elements-test.js

```diff
@@ -335,11 +335,11 @@ describe('Trace Elements gatherer - Animated Elements', () => {
     expect(animationTraceElements).toHaveLength(2);
     expect(animationTraceElements[0].animations).toEqual([
       {failureReasonsMask: 8224, unsupportedProperties: ['width']},
-      {name: 'alpha', failureReasonsMask: 8224, unsupportedProperties: ['height']},
-      {name: 'beta', failureReasonsMask: 8224, unsupportedProperties: ['font-size']},
+      {name: 'beta', failureReasonsMask: 8224, unsupportedProperties: ['height']},
+      {name: 'gamma', failureReasonsMask: 8224, unsupportedProperties: ['font-size']},
     ]);
     expect(animationTraceElements[1].animations).toEqual([
-      {name: 'gamma', failureReasonsMask: 0, unsupportedProperties: undefined},
+      {name: 'alpha', failureReasonsMask: 0, unsupportedProperties: undefined},
     ]);
```

Comment:
No actually. The animation names are all mixed up 😕 

Sorry I gave these animations shitty names when I first made this. `alpha` should be the animating height not `beta`. While you are here could you give these more descriptive names like `height-animation` instead of `alpha`.

I can help investigate why these animations are getting mixed up.

Reply:
Oh I think the animation ids just got mixed up (totally expected on regenerate) but we hardcode the id -> name mapping in this test (see L324)